### DISCUSSION TITLE
AppCenter logo T-shirt

### DISCUSSION THREAD

**Discussion by phase1geo** [elementary/store#111] [Product Ideas]
I would love to see an elementary OS AppCenter logo T-shirt for sale in the store.

**Comment by cassidyjames** [comment]
We previously had AppCenter tees and hoodies as an exclusive for an AppCenter crowdfunding campaign. It may have been long enough for that to not be an issue, but I wouldn't want to upset backers of that campaign by releasing it to everyone now—we might want to do a slightly different design so the original remains exclusive.
